src/Editor.js

```diff
@@ -1332,99 +1338,106 @@ const Editor = () => {
                   switch (input) {
                     case "short-string":
                       return (
-                        <div
-                          key={key}
-                          style={{
-                            display: "grid",
-                            gridTemplateColumns: "repeat(1, min-content 1fr)",
-                            alignItems: "center",
-                            justifyItems: "center",
-                            gap: 6,
-                            padding: 6
-                          }}
-                        >
-                          <Label htmlFor={`option-${key}`}>{label}</Label>
-                          <Input
-                            key={key}
-                            id={`option-${key}`}
-                            type="text"
-                            value={
-                              doc.layers.find(
-                                ({ id }) => id === [...view.selection][0]
-                              ).options[key]
-                            }
-                            onChange={({ currentTarget: { value } }) => {
-                              setState(current => ({
-                                ...current,
-                                doc: {
-                                  layers: current.doc.layers.map(layer =>
-                                    layer.id === [...view.selection][0]
-                                      ? {
-                                          ...layer,
-                                          options: {
-                                            ...layer.options,
-                                            [key]: value
-                                          }
-                                        }
-                                      : layer
-                                  )
-                                }
-                              }));
-                            }}
-                          />
-                        </div>
-                      );
-                    case "long-string":
-                      return (
-                        <div
-                          key={key}
-                          style={{
-                            display: "grid",
-                            gridTemplateColumns: "repeat(1, min-content 1fr)",
-                            alignItems: "center",
-                            justifyItems: "center",
-                            gap: 6,
-                            padding: 6
-                          }}
-                        >
-                          <Label htmlFor={`option-${key}`}>{label}</Label>
-                          <Textarea
+                        <>
+                          { displayErrorMessage ? <OptionsErrorMessage style={{ paddingTop: index === 0 ? "4px" : "0px", paddingLeft: "6px"}} >{error[0].message}</OptionsErrorMessage> : null }
```

Comment:
Given the changes above, this can just be:

```suggestion
                          { error ? <OptionsErrorMessage style={{ paddingTop: index === 0 ? "4px" : "0px", paddingLeft: "6px"}} >{error.message}</OptionsErrorMessage> : null }
```